go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/ethereum/go-ethereum
 
 go 1.20
 
+replace github.com/specularL2/specular/lib/el_golang_lib => ../../../lib/el_golang_lib
```

Comment:
Can we simply move the hooks into this repo. This works but feels super wonky. @ujvl @LEAFERx I'll let you chime in here.

Reply:
imo it would be nice to share the hooks across multiple go based clients in the future.
also i think we can get rid of this once the hooks lib is in the main branch